pkgs/development/compilers/open-watcom/v2.nix

```diff
@@ -120,7 +123,8 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://open-watcom.github.io";
     license = licenses.watcom;
-    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "x86_64-windows" "i686-windows" ];
+    platforms = with platforms; windows ++ unix;
+    badPlatforms = platforms.riscv ++ [ "powerpc64-linux" "powerpc64le-linux" "mips64el-linux" ];
```

Comment:
We don't have any hydra builders for those platforms and they have a rather low support tier, so I wouldn't be surprised if not many of the dependencies already fail to build.